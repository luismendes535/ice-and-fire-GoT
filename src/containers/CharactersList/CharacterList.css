.Container {
  display: flex;
  justify-content: center;
  flex-direction: column;
  margin: 0rem 5rem 0 5rem;
  width: 100%;
}
