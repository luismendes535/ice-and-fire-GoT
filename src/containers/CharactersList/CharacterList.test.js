import React from "react";
import ReactDOM from "react-dom";
import { shallow } from "enzyme";
import CharacterList from "./CharactersList";

describe("for CharacterList", () => {
  // let wrapper;
  // const mockFn = jest.fn();
  // beforeEach(() => {
  //   wrapper = shallow(<CharacterList />);
  // });
  it("should render properly", () => {
    // expect(wrapper).toMatchSnapshot();
  });
});
