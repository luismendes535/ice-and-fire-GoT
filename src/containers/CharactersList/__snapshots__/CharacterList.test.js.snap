// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`for CharacterList should render properly 1`] = `undefined`;
